### ISSUE TITLE
String data in variable.

### ISSUE THREAD

**Issue by MYeager1967** [bblanchon/ArduinoJson#260]
Is it possible yet to add string data from a variable to a key? The code in question is as follows.

key1 ["Error"] = message;  and adds {"Error":true} instead of the string data contained in the variable message. I know this was an issue way over a year ago but I figured with the newest version, it's had to have been fixed. I'm probably doing something wrong and would love a kick in the right direction.

Thank you,

Mike

**Comment by bblanchon** [comment]
Hi Mike,

This is fixed since [v5.0](https://blog.benoitblanchon.fr/arduino-json-v5-0/).
The most likely explanation is that you're using an older version.

**Comment by MYeager1967** [comment]
I just downloaded the new version from Github yesterday. Had to figure out how to change a few things over from the way they were done before (or at least the way I was doing them). Still have a few things to figure out. Is it possible that parts of the old library were cached somewhere and still being used? I had renamed the old library but now I’ve removed it completely. I’m going to run a few more tests to see if it works. If not, I can post the routine in question if you like.

 

Great library!!!
